src/DSharp.Compiler/ScriptModel/Symbols/GenericParameterSymbol.cs

```diff
@@ -1,11 +1,11 @@
-// GenericParameterSymbol.cs
+﻿// GenericParameterSymbol.cs
 // Script#/Core/Compiler
 // This source code is subject to terms and conditions of the Apache License, Version 2.0.
 //
 
 namespace DSharp.Compiler.ScriptModel.Symbols
 {
-    internal sealed class GenericParameterSymbol : TypeSymbol
+    internal sealed class GenericParameterSymbol : ClassSymbol
```

Comment:
Why the change to class symbols only? Can't remember if Structs go under class - Have a feeling they do, but unsure :D

Reply:
yeah, a emulated generic type is a class in the sense that it can have a single base class [0..1] and [0..N] interfaces